.env-template

```diff
@@ -2,4 +2,18 @@ COINMARKET_API_KEY=
 LISTINGS_LATEST_URL=
 BUCKET=
 CLOUD_STORAGE=
-BIGQUERY=
\ No newline at end of file
+BIGQUERY=
+PROJECT_ID=
+DATASET=
+DAY_DIM=
+MONTH_DIM=
+DATE_DIM=
+QUOTE_DIM=
+TAG_DIM=
+NAME_TAG=
+NAME_DIM=
+PRICE_FACT=
+TRADING_VOLUME_DAY_FACT=
+SUPPLY_FACT=
+RANK_FACT=
+INITIAL_LOAD=FALSE
```

Comment:
any other defaults?

Reply:
I need to actually populate the constant default values here. I will work on that.